Binary files modxslt/mod_xslt-1.1.tar and modxslt2/mod_xslt-1.1.tar differ
diff -urN modxslt/mod_xslt.c modxslt2/mod_xslt.c
--- modxslt/mod_xslt.c	Wed Apr 24 22:27:15 2002
+++ modxslt2/mod_xslt.c	Wed Apr 24 22:27:47 2002
@@ -251,7 +251,7 @@
   /* Setup some common strings of text, for file not found, etc. */
  
   regmatch_t pmatch[2];
-  regex_t *cpat = ap_pregcomp(r->pool, "\\<!DOCTYPE", REG_EXTENDED);
+  regex_t *cpat = ap_pregcomp(r->pool, "<!DOCTYPE", REG_EXTENDED);
 
   int rc = OK;
   int se = 1;
